schemas/arb.json

```diff
@@ -6,12 +6,292 @@
+                                                {
+                                                    "const": "yQQQ",
+                                                    "description": "YEAR_ABBR_QUARTER"
+                                                },
+                                                {
+                                                    "const": "yQQQQ",
+                                                    "description": "YEAR_QUARTER"
+                                                },
+                                                {
+                                                    "const": "H",
+                                                    "description": "HOUR24"
+                                                },
+                                                {
+                                                    "const": "Hm",
+                                                    "description": "HOUR24_MINUTE"
+                                                },
+                                                {
+                                                    "const": "Hms",
+                                                    "description": "HOUR24_MINUTE_SECOND"
+                                                },
+                                                {
+                                                    "const": "j",
+                                                    "description": "HOUR"
+                                                },
+                                                {
+                                                    "const": "jm",
+                                                    "description": "HOUR_MINUTE"
+                                                },
+                                                {
+                                                    "const": "jms",
+                                                    "description": "HOUR_MINUTE_SECOND"
+                                                },
+                                                {
+                                                    "const": "m",
+                                                    "description": "MINUTE"
+                                                },
+                                                {
+                                                    "const": "ms",
+                                                    "description": "MINUTE_SECOND"
+                                                },
+                                                {
+                                                    "const": "s",
+                                                    "description": "SECOND"
+                                                }
+                                            ],
+                                            "description": "The formatters are the ones provided by the DateFormat class in Dart's package:intl."
```

Comment:
nit: formatters -> formats?

Reply:
No, this actually means formatters, as in the method which formats